CONTRIBUTING.md

```diff
@@ -1,344 +1,507 @@
+aren't overlapping with work that's currently underway and that everyone is on
+the same page with the goal of the work you're going to carry out.
 
 #### 2. [Fork](https://help.github.com/articles/fork-a-repo/) [this repository](https://github.com/bids-standard/bids-specification) to your profile
 
-This is now your own unique copy of the BIDS Specification. Changes here won't affect anyone else's work, so it's a safe space to explore edits to the specification!
+This is now your own unique copy of the BIDS Specification. 
+Changes here won't affect anyone else's work, so it's a safe space to explore edits to the
+specification!
 
-Make sure to [keep your fork up to date](https://help.github.com/articles/syncing-a-fork/) with the master repository, otherwise you can end up with lots of dreaded [merge conflicts](https://help.github.com/articles/about-merge-conflicts/).
+Make sure to
+[keep your fork up to date](https://help.github.com/articles/syncing-a-fork/)
+with the master repository, otherwise you can end up with lots of dreaded
+[merge conflicts](https://help.github.com/articles/about-merge-conflicts/).
 
 #### 3. Make the changes you've discussed
 
-Try to keep the changes focused. If you submit a large amount of work in all in one go it will be much more work for whomever is reviewing your pull request. Please detail the changes you are attempting to make.
+Try to keep the changes focused. 
+If you submit a large amount of work in all in one go it will be much more work 
+for whomever is reviewing your pull request.
+Please detail the changes you are attempting to make.
 
 #### 4. Submit a [pull request](https://help.github.com/articles/about-pull-requests/)
 
 Please keep the title of your pull request short but informative - it will
 appear in the [changelog](src/CHANGES.md).
 
 Use one of the following prefixes in the title of your pull request:
-  - `[ENH]` - enhancement of the specification that adds a new feature or
+
+-   `[ENH]` - enhancement of the specification that adds a new feature or
     support for a new data type
-  - `[FIX]` - fix of a typo or language clarification
-  - `[INFRA]` - changes to the infrastructure automating the specification
+-   `[FIX]` - fix of a typo or language clarification
+-   `[INFRA]` - changes to the infrastructure automating the specification
     release (for example, building HTML docs)
-  - `[MISC]` - everything else including changes to the file listing
+-   `[MISC]` - everything else including changes to the file listing
     contributors
 
-If you are opening a pull request to obtain early feedback, but the changes
-are not ready to be merged (a.k.a. Work in Progress pull request) please
-use a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+If you are opening a pull request to obtain early feedback, but the changes are
+not ready to be merged (a.k.a. Work in Progress pull request) please use a
```

Comment:
a.k.a. sounds like a candidate to the "to be avoided list" :-) (etc ie eg)

Also, why the capitalization "Work in Progress"?